src/main/telemetry/frsky.c

```diff
@@ -364,20 +348,22 @@ static void sendVario(void)
     sendDataHead(ID_VERT_SPEED);
     serialize16(getEstimatedVario());
 }
+#endif
 
 /*
  * Send voltage via ID_VOLT
  *
  * NOTE: This sends voltage divided by batteryCellCount. To get the real
  * battery voltage, you need to multiply the value by batteryCellCount.
  */
-static void sendVoltage(void)
+static void sendVoltageCells(void)
 {
-    static uint16_t currentCell = 0;
+    static uint16_t currentCell;
```

Comment:
Variable declarations should be done right where they are used for the first time, and not at the begin of the function.

Reply:
This variable is `static`, it may be better to keep it clearly visible ..

Reply:
I agree with @ledvinap , for static variables it is generally best to put them at the beginning of the function.

Reply:
Ok. I normally try to keep them close to where they are used, which in my opinion improves readability. But I am good either way.

Reply:
Done, static at top and the rest when is used